AutoRest/Generators/CSharp/CSharp/ClientModelExtensions.cs

```diff
@@ -132,6 +132,36 @@ private static bool ShouldValidate(this IType model)
         }
 
         /// <summary>
+        /// Format the documentation of a property properly with the correct getters and setters. Note that this validation will
+        /// only acryonyms and article words.
```

Comment:
Minor issue: looks like the last sentence of this method summary has a missing word and typo (acronyms).